### PR TITLE
[Distributed] Fix a bug in graph partition.

### PR DESCRIPTION
**Pull request by zheng-da**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

**Comment by VoVAllen** [comment]
To trigger regression tests: 
 - `@dgl-bot run [instance-type] [which tests] [compare-with-branch]`; 
 For example: `@dgl-bot run g4dn.4xlarge all dmlc/master` or `@dgl-bot run c5.9xlarge kernel,api dmlc/master`

File: tests/distributed/test_partition.py [hunk 1]
```diff
@@ -120,24 +120,50 @@ def verify_hetero_graph(g, parts):
...
+        etype_ids, inner_type_eids = gpb.map_to_per_etype(inner_eids)
+        partid = gpb.eid2partid(inner_type_eids, etype)
+        assert np.all(F.asnumpy(etype_ids) == etype_id)
+        assert np.all(F.asnumpy(partid) == gpb.partid)
+
+        orig_id = F.boolean_mask(part.edata['orig_id'], inner_edge_mask)
+        local_eids = gpb.eid2localeid(inner_type_eids, gpb.partid, etype)
+
+        for name in g.edges[etype].data:
+            if name in [dgl.EID, 'inner_edge']:
```
Comment (classicsong) [inline tests/distributed/test_partition.py:156]: Shall we put inner_edge as dgl.INNER_EDGE?

**Review by classicsong — APPROVED** [review]
LGTM

File: tests/distributed/test_partition.py (hunk 1 above)
Comment (zheng-da) [reply tests/distributed/test_partition.py:156]: i'm not sure.

**Pull request merged**